deselected_tests.yaml

```diff
@@ -0,0 +1,14 @@
+deselected_tests:
+  # Deselecting 5 SVC related tests where duplicate samples end up being support vectors
+  # See: https://github.com/scikit-learn/scikit-learn/issues/12738
+  - svm/tests/test_svm.py::test_sample_weights
+  - svm/tests/test_svm.py::test_precomputed
+  - svm/tests/test_sparse.py::test_sparse
+  - svm/tests/test_sparse.py::test_svc_iris
+  - ensemble/tests/test_bagging.py::test_sparse_classification
+  # test_k_means_fit_predict is known to sporadically fail for float32 inputs in multithreaded runs
+  # See: https://github.com/IntelPython/daal4py/issues/25
+  - cluster/tests/test_k_means.py::test_k_means_fit_predict
+  # test_non_uniform_strategies fails due to differences in handling of vacuous clusters after update
+  # See https://github.com/IntelPython/daal4py/issues/69
+  - preprocessing/tests/test_discretization.py::test_nonuniform_strategies
```

Comment:
Going forward we probably need to deselect tests depending on the sklearn version.

Reply:
sklearn versioning in the yaml was implemented.